file.go

```diff
@@ -29,6 +29,19 @@ func NewFile(opts ...FilePropertyFunc) *File {
 	return f
 }
 
+// SetValidation stores ValidateOpts on the FEDWireMessage's validation rules
+func (f *File) SetValidation(opts *ValidateOpts) {
+	if opts == nil {
+		return
+	}
+	f.FEDWireMessage.ValidateOptions = opts
```

Comment:
Adam's comment below was never addressed. In this function and `GetValidation()`, we should be checking `if f == nil { // return (nil) }`.